### DISCUSSION TITLE
prisma equivalent to Mysql WHERE condition columns comparison in same table?

### DISCUSSION THREAD

**Discussion by ghost** [prisma/prisma#7178] [Q&A] [answered]
Hey there, I'm using  "@prisma/client": "^2.21.2", 
Assumning below model

```prisma
model EDI_Price {
  CUSIP		        String
  PrimaryExchange	String
  Exchange		String
  Currency		String
  Price			Float
  PriceDate		Date
 }
```

How can I do WHERE condition to compare in columns in same table.
raw sql example:
`SELECT * FROM EDI_Price WHERE PrimaryExchange=Exchange AND CUSIP='594918104';`

I tried something like below:
```ts
const data = await prisma.EDI_Price.findMany({
   where: {
        CUSIP: '594918104',
         PrimaryExchange: { 
                equals: { Exchange }
          },
    }
})
```

but it didn't work as for prisma Exchange is a variable, whose value is undefined. And hence throw an error 'Exchange' is not defined.

can we compare columns in prisma?

**Comment by ryands17** [comment, marked as answer]
@manishzversal 👋 

Unfortunately not, but we do have a feature request for the same: https://github.com/prisma/prisma/issues/5048

It would be great if you could add a 👍 to the issue so that we can look at the priority.

**Reply by ghost** [reply]
Thank you @ryands17  for quick response. So for the time being prisma raw query is the way to go for my issue?

**Reply by ryands17** [reply]
Yes that would be the way to go: https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access
